packages/fluent-editor/src/table/modules/table-column-tool.ts

```
@@ -153,14 +152,14 @@ export default class TableColumnTool {
       // IE11/Edge 下 height 会变成 'auto'
-      let computedHeight = row && getComputedStyle(row)['height']
+      let computedHeight = row && getComputedStyle(row).height
       if (computedHeight === 'auto') {
         computedHeight = row.querySelector('td').style.height || '30px'
       }
 
-      let rowHeight = row && parseFloat(computedHeight)
+      let rowHeight = row && Number.parseFloat(computedHeight)
       if (rowHeight < CELL_MIN_HEIGHT) {
-        const rowChildHeight = row && row.childNodes[0] && parseFloat(getComputedStyle(row.childNodes[0])['height'])
+        const rowChildHeight = row && row.childNodes[0] && Number.parseFloat(getComputedStyle(row.childNodes[0]).height)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)